livefs_edit/context.py

```diff
@@ -197,26 +207,65 @@ def teardown(self):
             run(['mount', '--make-rprivate', mount])
             run(['umount', '-R', mount])
         shutil.rmtree(self.dir)
+        for loop in reversed(self._loops):
+            run(['losetup', '--detach', loop])
 
-    def mount_iso(self):
-        self._iso_overlay = self.add_overlay(
-            self.add_mount(
-                'iso9660', self.iso_path, self.p('old/iso'),
-                options='loop,ro'),
-            self.p('new/iso'))
-
-    def repack_iso(self, destpath):
+    def find_livefs(self, device):
+        for dev in glob.glob(f'{device}*'):
+            try:
+                try_mount = self.add_mount(None, dev, None, options='ro')
+            except subprocess.CalledProcessError:
+                continue
+            try:
+                if glob.glob(try_mount.p('casper/*.squashfs')):
```

Comment:
Ok, I like this: simple but 'just works'. When preparing my own branch I was thinking that doing guesswork on where the live-media fs is would be a bit too involving, but basically just looking for a partition with casper squashfses indeed is enough!

Reply:
It's what casper does! (Well, more or less -- casper supports some non-squashfs files too but blah).